composer.json

```diff
@@ -22,15 +22,18 @@
     "require": {
         "php": "^7.1",
         "doctrine/data-fixtures": "^1.3",
-        "doctrine/doctrine-bundle": "^1.11|^2.0",
+        "doctrine/doctrine-bundle": "^1.11 || ^2.0",
```

Comment:
I'm not too keen on changing this syntax here since we use single-pipe in all other repos. Mind changing it back? I'm happy to revisit this in a separate PR.

Reply:
Even though it's the official notation (using double pipes), I changed it back.